src/Avalonia.Styling/Styling/Style.cs

```diff
@@ -175,4 +180,8 @@ private static void ControlDetach(IStyleable control)
             _applied.Remove(control);
         }
     }
+
+    public class StyleResources : Dictionary<string, object>
```

Comment:
👍  It's a good idea to use a class for this rather than directly using a `Dictionary<string, object>`.

However seeing as you've done this, could you:

1. Move it to a separate file
2. Make it not inherit from `Dictionary` but instead implement `IDictionary<string, object>` and use composition to defer to an inner `Dictionary<string, object>`? That way we can be free to change the implementation should we need to.